Source/ThirdParty/ImGuiLibrary/Include/imconfig.h

```diff
@@ -104,7 +104,7 @@
 // Your renderer backend will need to support it (most example renderer backends support both 16/32-bit indices).
 // Another way to allow large meshes while keeping 16-bit indices is to handle ImDrawCmd::VtxOffset in your renderer.
 // Read about ImGuiBackendFlags_RendererHasVtxOffset for details.
-#define ImDrawIdx unsigned int
+//#define ImDrawIdx unsigned int
```

Comment:
This should remain uncommented to keep supporting 32-bit vertex indices.

Reply:
uncommented